transcom/mymove
# B 19030 int

## [B-19030](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19030)

## Summary

This is a very simple 1-line change to adjust the verbiage associated with the "Edited" status when approving PPM advances. For this story, all that was required was for the display text to be altered, as the Edited status already functions as an approval. Additionally, it's important to keep the status as 'Edited' as it functions differently from the approved status in that allows the API to adjust the approved value when the call is made.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Access the application as a customer
2. Create and submit a PPM with an advance request
3. Login as a Service Counselor
4. Edit the shipment
5. Adjust the Advance request value on the second page. Approve the advance request.
6. Ensure the details page shows "Approved" instead of "Approved with adjustment"
<img width="509" alt="image" src="https://github.com/transcom/mymove/assets/148906585/3ea35657-c57b-42d8-9379-38695458a94c">

7. Ensure all client and playwright tests pass.


### Frontend
